packages/flutter/test/widgets/media_query_test.dart

```diff
@@ -14,7 +14,15 @@ void main() {
     await tester.pumpWidget(
       new Builder(
         builder: (BuildContext context) {
-          size = MediaQuery.of(context).size;
+          final MediaQueryData data = MediaQuery.of(context);
+          expect(data.toString(), hasOneLineDescription);
```

Comment:
I wonder if that was required in the past.  I'll update all the other call sites too.